deploy-deps.sh

```diff
@@ -44,6 +46,15 @@ deploy() {
     deploy_registry
 }
 
+test_pvc_binding(){
+    local pvc_resources="${script_path}/dependencies/pre-deployment-pvc-binding"
+    echo "Creating PVC from '$pvc_resources' using the cluster's default storage class"
+    kubectl apply -k "$pvc_resources"
+    kubectl wait --for=jsonpath='{status.phase}'=Bound pvc/test-pvc -n test-pvc-ns --timeout=20s
```

Comment:
Can we use retry here? If we merge the PR that allows adding message to retry then we can make use of it to make this failure clearer.

Reply:
Sorry, I left it out by mistake. yes, retry is intended to be used.
fixed